README.md

```diff
@@ -76,15 +76,15 @@ inspector = GhInspector::Inspector.new "fastlane", "fastlane"
 inspector.search_query "Someone set us up the bomb", FastlaneUI.new
 ```
 
-
 Protocol for custom objects:
 
  - `inspector_started_query(query, inspector)` - Called just as the investigation has begun.
+ - `inspector_successfully_recieved_report(report, inspector)` - Deprecated: Please use `inspector_successfully_received_report` instead.
```

Comment:
Didn't find a way to make `yard` ignore these, but maybe it's for the better. If an old user comes back to this they'll see that they got deprecated.